source/scripts/modules/Keys.js

```diff
@@ -13,19 +13,12 @@ window.uiGradients = window.uiGradients || {};
 
       $(document).on('keydown', function(event) {
         switch(event.which) {
-
+          
           case 37: // left
-            if (pallete.hasClass('is-active')) {
-              return;
-            }
-            canvas.trigger('changeDown');
-          break;
-
           case 39: // right
-            if (pallete.hasClass('is-active')) {
-              return;
-            }
-            canvas.trigger('changeUp');
+            if (pallete.hasClass('is-active')) return;
+
+            (event.which === 37 ) ? canvas.trigger('changeDown') : canvas.trigger('changeUp');
```

Comment:
No problem, its called [fallthrough](http://stackoverflow.com/questions/188461/), give it a read :)